drf_hal_json/pagination.py

```diff
@@ -16,6 +17,10 @@ def get_paginated_response(self, data):
             links['next'] = {'href': self.get_next_link()}
         if self.get_previous_link():
             links['previous'] = {'href': self.get_previous_link()}
+        template_url = replace_query_param(self.request.build_absolute_uri(), self.page_query_param, '_PAGE_')
+        links['pages'] = {
```

Comment:
I understand the use case now. I think this should be `links['page']`, since it is a link to a specific page.